analysis/viewshed-geoelement/README.md

```diff
@@ -1,29 +1,39 @@
-# Viewshed GeoElement
+# Viewshed geoelement
 
-Attach a viewshed to an object to visualize what it sees.
+Analyze the viewshed for an object (GeoElement) in a scene.
 
-![](ViewshedGeoElement.gif)
+![Image of viewshed for geoelement](ViewshedGeoElement.png)
+
+## Use case
+
+A viewshed analysis is a type of visual analysis you can perform on a scene. The viewshed aims to answer the question 'What can I see from a given location?'. The output is an overlay with two different colors - one representing the visible areas (green) and the other representing the obstructed areas (red).
 
 ## How to use the sample
 
-Once the scene is done loading, click on a location for the tank to drive to. It will automatically turn and drive straight towards the clicked point. The viewshed will automatically move and rotate with the tank.
+Click to set a destination for the vehicle (a GeoElement). The vehicle will 'drive' towards the tapped location. The viewshed analysis will update as the vehicle moves.
```

Comment:
```suggestion
Click to set a destination for the vehicle (a GeoElement). The vehicle will 'drive' towards the clicked location. The viewshed analysis will update as the vehicle moves.
```

Reply:
Even when I thought I had them all! Thanks for catching this one. For the next batch I'll get in the habit of searching each file for 'tap'.